### DISCUSSION TITLE
(commit vs branch) + (commit menu -> revert changes should've be named undo changes)

### DISCUSSION THREAD

**Discussion by PowerfulShell** [community/community#51067] [New to GitHub]
(commit vs branch)
create branch in that specific commit, is like bottom up to that "commit #"
more like creating a version.

--------------------

 (commit menu -> revert changes should've be named undo changes)
in the history / commit list, it is (((wrongful!!!))) misleading, to name this menu 
"revert changes in commit" 
should have been
"undo this commit"

because it is illogical to use the word "revert" it gives a hint of going back to this point of commit.....
i have waste a good hour on how to use github desktop due to this issue...
NOW
if I want to keep a good working version from somebody, 
first i fork.
second i use github desktop to clone it locally,
third I will sync my fork with "official / original" repo,
so I then can fetch from my own repo to update, 
AND
mange my versioning via git deksotp

--------------

back to the first topic 
(commit vs branch) 
so when they update you find it broken you want to roll back to previous serval state, 
you dont go each one of the commit and hit that revert thing.... thats what GOOGLE and GITHUB doc tells me.....
correct way is in that working commit you branch it, :)

just my 2cents
noob to git

**Comment by queenofcorgis** [comment]
Thanks for the feedback, I've added the proper label on this so the Desktop team can easily review it. Feel free to share any more details!

**Comment by PowerfulShell** [comment]
yes, thanks a lot!

i use github desktop as version control and can try out different new things.

if what you want to do is creating different version control, then you would need to do is:
"create branch from commit"
not
"revert change in commit"

when i search google how to go back to certain point of commit, both Google and GitHub doc pointed to the wrong direction .....LOL
in the doc it say just "revert change in commit"

it should simply be "create branch from commit"

better yet, give the newly branch a commit hash name as default option rather than needing you to type it in manually....

from my point of view,
"revert change in commit" isnt good phrasing,  
"revert change in this commit" seem too long.....
"revert this change" seemed good enough, as the list is list of commit, no need to say commit twice...

the concept of "branch" naming, is kind of TOO BIG, really over shooting....LOL it will give outsider lost of translation.....
"create branch from commit" may sound ambiguous at first glance, like it is a BIG JOB to DO .....LOL 
more like "create version from here", "rollback to this commit"..... why rollback you say? because the latest would be "master" :) LOL

so 

latest is master,
IF 
create from a set point from certain commit, 
then
it should be named version, rather than branch, (i dont hate branch i think it good naming, but in this case it is a bit confusing and sound TOO BIG) 
because most of the time we arent making new progress as non-coder, and noob coder likewise.....

thank you

**Comment by PowerfulShell** [comment]
maybe not to change too much, just add a hint to the existing menu, that would clear out a lot.

and
